test_runner/src/main/kotlin/ftl/args/IosArgs.kt

```diff
@@ -33,7 +33,7 @@ IosArgs
       record-video: $recordVideo
       timeout: $testTimeout
       async: $async
-      client-details: ${ArgsToString.mapToString(clientDetails)}
+      client-details:${ArgsToString.mapToString(clientDetails)}
```

Comment:
hmmm space was removed but in other places above it includes a space. Is there a reason for this change?

Reply:
Yeah, that is tricky, I removed extra space before ${...} only because if map/list is empty function returns an empty string, as result, we end up with extra space at the end of line. That is extremely annoying when you create regex.... Places with `$[value]` has either null or default value